manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml

```diff
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/tools#tools-platform
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android SDK Platform-Tools
+PackageUrl: https://developer.android.com/tools/releases/platform-tools
+License: Apache License
+LicenseUrl: https://android.googlesource.com/platform/packages/modules/adb/+/refs/tags/platform-tools-35.0.1/NOTICE
+Copyright: Copyright (c) 2006-2009, The Android Open Source Project; Copyright 2006, Brian Swetland <swetland@frotz.net>
+CopyrightUrl: https://android.googlesource.com/platform/packages/modules/adb/+/refs/tags/platform-tools-35.0.1/NOTICE
+ShortDescription: |-
+  Android SDK Platform-Tools is a component for the Android SDK.
+  It includes tools that interface with the Android platform, primarily adb and fastboot.
+Tags:
+- adb
+- android
+- debug
+- development
+- fastboot
+- flash
+- platform-tools
+Documentations:
+- DocumentLabel: Android Debug Bridge (adb)
+  DocumentUrl: https://developer.android.com/tools/adb
+ManifestType: defaultLocale
+ManifestVersion: 1.9.0

```

Comment:
```suggestion
ManifestVersion: 1.7.0
```

Reply:
Manifest 1.9.0 is [live](https://github.com/microsoft/winget-pkgs/pull/195988).